### ISSUE TITLE
ELI5 way to set it up please?

### ISSUE THREAD

**Issue by aensaro** [SyntaxColoring/Markov-Word-Generator#1]
Hi, I've used the page you set up but I'd like to try it locally as well. I installed node.js and typing
`coffee -v`
in cmd shows me I installed it correctly.
But now what?
How can I load a dictionary file? How can I generate a word or a list of words?
I tried reading the readme multiple times but being completely new to node.js and coffee scripts I don't know what to do at all. 
I tried opening the local index.htm but it doesn't work...

**Comment by SyntaxColoring** [comment]
If the page isn't working when you open it locally, I think the problem is with the way it includes its CoffeeScript sources.  When the whole project is stored locally, the included CoffeeScript compiler will try to access the local filesystem (because that's where the CoffeeScript sources are).  Chrome sees this as a security issue and forbids it.  If you open the Chrome dev console, you'll see cross-origin errors like this:

    XMLHttpRequest cannot load file:///some-path/Markov-Word-Generator/markov.coffee. 

There are a couple ways of working around this.

1. Disable the browser's security check.  Launching Chrome with `google-chrome --allow-file-access-from-files index.html` seems to work.
2. Use a browser that allows this kind of local file access by default, like Firefox.
3. Compile the CoffeeScript yourself so no local file access is needed.  Run something like `coffee -c markov.coffee word-generator.coffee`.  Then, modify `index.html` to remove `coffee-script.js`, and to include `markov.js` and `word-generator.js` instead of `markov.coffee` and `word-generator.coffee`.

Option 3 is probably what I should have been doing in the first place.  When I made this project, though, it seemed like an unnecessary complication.

Does this answer your question?

**Comment by aensaro** [comment]
It works in Firefox, awesome. Thank you!
